app/pb_kits/playbook/pb_date_range_inline/_date_range_inline.jsx

```diff
@@ -25,41 +33,112 @@ const dateTimeIso = (dateValue) => {
 }
 
 const DateRangeInline = (props: DateRangeInlineProps) => {
-  const { endDate, startDate } = props
+  const {
+    icon = false,
+    dark = false,
+    size = 'sm',
+    align = 'left',
+    startDate,
+    endDate,
+  } = props
+
+  const iconContent = () => {
+    return (
+      <If condition={icon}>
+        <Body
+            color="light"
+            tag="span"
+        >
+          <Icon
+              className="pb_date_range_inline_icon"
+              dark={dark}
+              fixedWidth
+              icon="calendar-alt"
+              size={size}
+              tag="span"
+          />
+        </Body>
+      </If>
+    )
+  }
+
```

Comment:
```suggestion
const dateRangeClasses =  buildCss("pb_date_range_inline_kit_", align)
```

Reply:
I gotta check this real quick.

Reply:
I have not tested this locally to verify.. just fyi. But this is kinda the pattern we typically try for.  It allows us to add more objects in a cleaner and more separated way, if the component needs to expanded down the road.